test/pact/contTXOUTNew.pact

```diff
@@ -25,9 +25,9 @@
       }
     , "events": []
     , "gas": 6
-    , "logs": "9JwtbUnJnZVC62lihy1YDA7Gmi2Mkopl62GkMGB3QZI"
+    , "logs": "c_Xbqb7BZetrHBtU7nGdZ4yoc1_Y1iE3CgsO03gOrbc"
     , "meta": {}
-    , "req-key": "rwFh_qoxumclpxZj6QIo5GOM0CmE5AztlKukhFCRTVA"
+    , "req-key": "pHsX8RJkJNRlrGflD_vRiKOsjlLSVGdf1hYU_eI6fgU"
```

Comment:
Note: these req keys and logs changed because we added `NetworkId` to the command.